packages/server/services/ReactionSevice.ts

```diff
@@ -0,0 +1,41 @@
+import { IReaction, ReactionModel } from '../models';
+import { BaseRESTService } from './BaseRESTService';
+
+type WithIdRequest = {
+  id: number;
+};
+
+class ReactionService implements BaseRESTService {
+  async find({ id }: Partial<WithIdRequest>) {
+    if (id) {
+      return await ReactionModel.findOne({
+        where: {
+          id,
+        },
+      });
+    }
+    return await ReactionModel.findAll();
```

Comment:
Ключевое слово await перед return-ом ненужный и может усложнить дебаггинг, подробнее тут https://eslint.org/docs/latest/rules/no-return-await

Достаточно просто:
```suggestion
    return ReactionModel.findAll();
```